canonical/microk8s.io
# Apply landing page redesign

## Done

- Applied redesign as per [doc](https://docs.google.com/document/d/1ObWmhRc4GCIhrBi8gT7yIIs0J_dn_8ravfLtcwhr3uQ/edit) and [design](https://www.figma.com/file/0DTIS3uNNUuDAdCKWrk8Uf/23.10---Microk8s.io?node-id=0%3A1&mode=dev)

**NOTE: Nav bar and footer to be updated in coming PR**

## QA

- View the site locally in your web browser at: https://microk8s-io-610.demos.haus/
- Check against doc and design


## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3755
